.github/workflows/release.yml

```diff
@@ -36,39 +36,90 @@ env:
 jobs:
   release-build:
     timeout-minutes: 45
-    runs-on: [ubuntu-ghcloud]
     strategy:
+      matrix:
+        os: [windows-ghcloud]
       fail-fast: false
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Clean up tag name
         shell: bash
-        run: |
+        run: | 
           echo "sui_tag=$(echo ${{ env.TAG_NAME }} | sed s/'refs\/tags\/'//)" >> $GITHUB_ENV
 
       - name: Checking out ${{ env.sui_tag }}
         if: ${{ env.sui_tag != 'main' }}
         uses: actions/checkout@v3
         with:
           ref: ${{ env.sui_tag }}
+      
+      - name: Install nexttest
+        if: ${{ matrix.os == 'windows-ghcloud' && env.sui_tag != 'main'  }}
+        uses: taiki-e/install-action@nextest
+
+      - name: Setup protoc
+        if: ${{ matrix.os == 'windows-ghcloud' && env.sui_tag != 'main'  }}
+        uses: arduino/setup-protoc@v1
+        # this avoids rate-limiting
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install postgres (Windows)
+        if: ${{ matrix.os == 'windows-ghcloud' && env.sui_tag != 'main'  }}
+        shell: bash
+        run: |
+          choco install postgresql12 --force --params '/Password:root'
```

Comment:
Nothing newer than Pg12?  We're using Pg 15 in prod.

Reply:
I copied this code from another workflow. So 🤷🏼

Reply:
See https://github.com/MystenLabs/sui/blob/main/.github/workflows/rust.yml#L156